app/services/recruitment_cycle_creation_service.rb

```diff
@@ -10,12 +10,12 @@ def initialize(year:, application_start_date:, application_end_date:)
   end
 
   def call
-    recruitment_cycle = RecruitmentCycle.create!(
+    RecruitmentCycle.create!(
       year: @year,
       application_start_date: @application_start_date,
       application_end_date: @application_end_date
     )
 
-    RolloverJob.perform_later(recruitment_cycle.id)
+    RolloverJob.perform_later
```

Comment:
🤔 I think it would be better to pass through the ID here, no?

Reply:
Also, if you were to create multiple years or many years ahead (not saying that a user should be able to do so, but based on the code), it would always refer to the current cycle.

Reply:
Then we would have to change the rollover code to handle this scenario. Can this be another PR and another trello card?

Reply:
Sure. :)